internal/engine/wazevo/backend/isa/amd64/machine.go

```diff
@@ -1918,6 +1918,9 @@ func (m *machine) lowerCall(si *ssa.Instruction) {
 		for i := regalloc.RealReg(0); i < 16; i++ {
 			m.insert(m.allocateInstr().asDefineUninitializedReg(regInfo.RealRegToVReg[xmm0+i]))
 		}
+		// Since Go 1.24 it may also use DX, which is not reserved for the function call's 3 args.
+		// https://github.com/golang/go/blob/go1.24.0/src/runtime/memmove_amd64.s#L123
+		m.insert(m.allocateInstr().asDefineUninitializedReg(regInfo.RealRegToVReg[rdx]))
```

Comment:
instead of uninitialized, but shouldn't we need to put the proper value on it? otherwise it seems to me it would behave unexpectedly

Reply:
maybe looking at the memmove signature change see what's the expected value on rdx

Reply:
(this makes me feel like this might not be the amd64 only issue)

Reply:
ah ok no i get it